### PR TITLE
CVE-2016-8747

### PR DESCRIPTION
**Pull request by mxm8856**

### CODE REVIEW

File: cves/CVE-2016-8747.yml [hunk 1]
```diff
@@ -43,7 +43,7 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: Http11InpouutBuffer.java allowed remote attackers to read data (information discloure) that was intended for a different request.
```
Comment (GORosin) [inline cves/CVE-2016-8747.yml]: How did this java file allow attackers to read data?

File: cves/CVE-2016-8747.yml [hunk 2]
```diff
@@ -141,7 +141,7 @@ interesting_commits:
     emerging themes?
     If there are no interesting commits, demonstrate that you completed this
     section by explaining what happened between the VCCs and the fix.
-  answer:
+  answer: false
```
Comment (GORosin) [inline cves/CVE-2016-8747.yml:153]: Were there any commits at all in the same subsystem between the VCC and fix?

File: cves/CVE-2016-8747.yml [hunk 3]
```diff
@@ -4,27 +4,27 @@ nickname_instructions: |
...
   alongside its CVE to make it more easily distinguished from the rest.
-nickname:
+nickname: Buffer Info Disclosure Mixup
 
 CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+CWE: 1230
```
Comment (GORosin) [inline cves/CVE-2016-8747.yml]: Upvotes: 3

File: cves/CVE-2016-8747.yml (hunk 2 above)
Comment (mxm8856) [reply cves/CVE-2016-8747.yml:153]: Upon initial research: no. I will double check.

File: cves/CVE-2016-8747.yml (hunk 1 above)
Comment (mxm8856) [reply cves/CVE-2016-8747.yml]: It wasn't specifically described in any article. However, from piecing articles together it seems like the data was exposed and not properly encrypted in the buffer.

**Pull request merged**